lib/Logger/LoggerFeature.h

```diff
@@ -81,6 +84,8 @@ class LoggerFeature final : public application_features::ApplicationFeature {
   bool _supervisor = false;
   bool _backgrounded = false;
   bool _threaded = false;
+  std::string _apiSwitch;
+  bool _apiEnabled;
```

Comment:
```suggestion
  bool _apiEnabled;
  std::string _apiSwitch;
```

Reply:
I prefer the old order, since the `_apiSwitch` comes first and then `_apiEnabled` is computed from it in `validateOptions`.